week1/4_Variables.ipynb

```diff
@@ -352,7 +352,7 @@
    "source": [
     "<p style=\"text-align: right; direction: rtl; float: right;\">\n",
     "קניתי 5 ארגזים גדולים של קרמבו. בארגז יש 10 קרמבו בכל שורה, ו־4 קרמבו בכל עמודה.<br>\n",
-    "נסו לשים כל אחד מהנתונים שמצאתם בשאלה במשתנה, ולהדפיס את התשובה הנכונה.\n",
+    "נסו לשים כל אחד מהנתונים שמצאתם בשאלה במשתנה, ולהדפיס כמה קרמבו יש לי.\n",
```

Comment:
Excellent. I usually fix myself when talking in general instead of mentioning specifics, but it's hard to find it all.